### PR TITLE
feat: :sparkles: Adding sections,: hero, programs, curriculum, misson…

### PR DESCRIPTION
**Pull request by JUAN-BH**
Adding some sections of our page.

### CODE REVIEW

File: footer.php
Comment (jcamiloguz) [inline footer.php:1]: To a dynamic year you can use
` <?php echo date("Y"); ?>`

File: style.css
Comment (jcamiloguz) [inline style.css:1]: BEM in form? :thinking:

**Review by jcamiloguz — CHANGES_REQUESTED** [review]

File: footer.php
Comment (JUAN-BH) [reply footer.php:1]: okey

File: style.css
Comment (JUAN-BH) [reply style.css:1]: i did it

**Review by jcamiloguz — APPROVED** [review]

**Pull request merged**